pages/linux/flameshot.md

```diff
@@ -0,0 +1,25 @@
+# flameshot 
+
+> Screenshot utility with a gui interface.
+> Supports basic image editing, such as; text,shapes,colors,imgur.
+> More information: <hhttps://flameshot.js.org/>.
+
+- Launch flameshot in gui mode:
+
+`flameshot launcher`
+
+- Click and drag screenshot:
```

Comment:
Perhaps this will read better?
```suggestion
- Take a screenshot by clicking and dragging:
```

Reply:
@sbrl I think it reads better, but I was debating if it is better, since when you glance quickly it is more evident. However I am a total noob so I am going to follow your lead on this.